src/duplicate-results/duplicate-results.tsx

```diff
@@ -41,7 +52,7 @@ export function DuplicateResults() {
 	if ( ! requestsWereMade || searchTerm === '' ) {
 		resultsContainerDisplay = (
 			<PlaceholderMessage
-				illustration={ InitialIllustration }
+				illustration={ <InitialIllustration /> }
```

Comment:
Ah good question! AFAIK, I don't think it makes too much a difference, but I'm not entirely sure... This case was more about consistency and getting the typing to play together nicely. I wanted to apply a class upriver to one of the illustration components, so that was the motivation to switching over to this syntax!